### DISCUSSION TITLE
【投票】你更喜欢哪种文档风格呢？

### DISCUSSION THREAD

**Discussion by ForteScarlet** [simple-robot/Discussions#182] [投票箱]
我们目前有两种风格的文档：
1. 🏠**v3版本官网** 使用的 [Docusaurus](https://www.docusaurus.io/zh-CN/) 风格文档（参考：[v3版本官网](https://simple-robot-library.github.io/simbot3-website/) ）
    - 优点：
        1. 样式更丰富，可定制化程度更高
        2. 基于 MDX 和 React，灵活度更高
        3. 有相对好理解、好实现的多版本功能
    - 缺点：
        1. 加载慢，而且在GitHub加持下是非常慢（单页面应用，定制化资源太多，比如有好几种字体，以及 Google 分析API）
        2. 相对比较重
        5. 源码维护相对更困难（因为代码提示不友好，且代码分析程度弱）
        6. 缺少一些实用功能（比如文档片段共享、变量插入、代码片段内增加超链接等）
2. 📕**大别野组件** 使用的 [Writersite](https://www.jetbrains.com/zh-cn/writerside/) 风格文档（参考：[大别野组件](https://simple-robot.github.io/simbot-component-miyoushe-villa/)）
    - 优点：
        1. 更好的代码提示与代码分析
        2. 更轻量化的文档源文件（基于 md 和 xml）
        3. 统一、清晰的文档布局，且很适合当文档的布局
        4. 有文档片段共享、变量插入、代码片段内增加超链接等 `Docusaurus` 默认没有的功能
    - 缺点：
        1. 处于 EAP 阶段，且文档（感觉）不够全面
        2. 定制化程度相对较低，比如我现在不知道怎么自定义字体
        3. 多版本功能有点儿云里雾里

**Comment by RoyilEs** [comment]
个人偏向于 轻量化和简洁明了的

**Reply by ForteScarlet** [reply]
🤔其实总的来说这俩都不算非常重，在其中的话我个人觉得 `Writersite` 会更轻一点儿
